content/kubeone/main/guides/cis-benchmarking/cis1.8-kubeone1.7-k8s1.27/_index.en.md

```diff
@@ -0,0 +1,574 @@
++++
+title = "Benchmark on KubeOne-1.7.3 and Kubernetes-1.27"
```

Comment:
We don't use hyphen to separate product name and version. Also, I'd consider rotating the Kubernetes version and the KubeOne version, and dropping the KubeOne patch release:
```suggestion
title = "Benchmark on Kubernetes 1.27 with KubeOne 1.7"
```